validation/validate_results.R

```diff
@@ -0,0 +1,160 @@
+# Validate benchmarking dataset results
+# Step 1: Ensure you have read the README in this directory and followed the instructions on creating the reference
+#         and test data
+# Step 2: Set tool_directory to the root directory of the DER disturbance analysis repo in your file system
```

Comment:
Is it possible to get tool_directory automatically? Potentially something along the lines of [https://stackoverflow.com/questions/1815606/determine-path-of-the-executing-script](url) ?

Reply:
It's a bit nasty, I've updated it to use the root directory by default with the option to override it if it detects that it's running somewhere else